File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        priority_queue<int,vector<int>,greater<int>>pq;
        for (const auto &item: nums){
            pq.push(item);
            if(pq.size()>k){
                pq.pop();
            }
        }
        return pq.top();
    }
};

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> mxh = new PriorityQueue<>((a, b)->Integer.compare(b,a));

        for(int i : nums ){
            mxh.add(i);
        }

        while(k-- > 1){
            mxh.poll();
        }

        return mxh.poll();
    }
}